spancontext.go

```diff
@@ -222,10 +217,9 @@ func (c *SpanContext) SetMessage(message MessageSpanContext) {
 // Both service.Name and service.Resource are required. If either is empty,
 // then SetDestinationService is a no-op.
 func (c *SpanContext) SetDestinationService(service DestinationServiceSpanContext) {
-	if service.Name == "" || service.Resource == "" {
+	if service.Resource == "" {
 		return
 	}
-	c.destinationService.Name = truncateString(service.Name)
```

Comment:
We still need to send this field to APM Server, as older versions (<7.14) require it. Seeing as we don't actually make use of the value, I think if it's unset we could just set it to `service.Resource`. Alternatively, see my comment about omitempty on the model type